worldview.spec

```diff
@@ -1,6 +1,6 @@
 Name: @WORLDVIEW@
 Version: @BUILD_VERSION@
-Release: @BUILD_RELEASE@.@BUILD_NUMBER@%{?dist}
+Release: 0.@BUILD_NUMBER@%{?dist}
```

Comment:
I'm curious why "0." here?

Reply:
Since we are both the developers and the packagers, it can stay at zero. It is a separate version number that can get bumped if the packaging needs an update when the version of the application hasn't changed. Current scripts were pulling that value from package.json but it appears that value was removed at some previous time. If we aren't going to use it, might as well be zero.